modules/azure/src/test/java/org/testcontainers/containers/CosmosDBEmulatorContainerTest.java

```diff
@@ -0,0 +1,44 @@
+package org.testcontainers.containers;
+
+import com.azure.cosmos.CosmosAsyncClient;
+import com.azure.cosmos.CosmosClientBuilder;
+import com.azure.cosmos.models.CosmosContainerResponse;
+import com.azure.cosmos.models.CosmosDatabaseResponse;
+import org.junit.Rule;
+import org.junit.Test;
+import org.testcontainers.utility.DockerImageName;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * @author Onur Kagan Ozcan
+ */
+public class CosmosDBEmulatorContainerTest {
+
+    @Rule
+    // emulatorContainer {
+    public CosmosDBEmulatorContainer emulator = new CosmosDBEmulatorContainer(
+        DockerImageName.parse(CosmosDBEmulatorContainer.LINUX_AZURE_COSMOS_DB_EMULATOR)
+    );
+    // }
+
+    @Test
+    // testWithEmulatorContainer {
```

Comment:
You can move the markers into the test method, I think this would like nicer in the docs. See [the Kafka module](https://www.testcontainers.org/modules/kafka/) as an example how it would look.

Reply:
👍 ah ok, let me try.